unit_tests/fake/fake_error_manager.cpp

```diff
@@ -31,7 +31,7 @@ _er_log_debug (const char *file_name, const int line_no, const char *fmt, ...)
 int
 er_errid (void)
 {
-  return 0;
+  return ER_FAILED;
```

Comment:
I need this function to return any error because this is how we're able to simulate a data page returned with error. We can chose not to link with this library (fake), but do we really want that? That's gonna be a lot of API to mock. Furthermore this is not used by other tests..

Reply:
It is not used now, but it is meant to be used by any future tests. The expected behavior in most cases will be to get no error. If another behavior is required in this test, better to avoid using the fake_error_manager.cpp file and add the definition in the main file of the test.

Reply:
Found a good solution with "er_set".